index.js

```diff
@@ -1,17 +1,90 @@
 class SortedList {
-  constructor() {}
+  constructor ()
+  {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add ( item )
+  {
+    this.items.push( item );
+    this.items.sort( function ( a, b )
+    {
+      return a - b;
+    } );
+    this.length = this.items;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (!this.items[pos]) {
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0) {
```

Comment:
this works, but remember that they also made you create a `length `property in the constructor